Vision/proposal.md

```diff
@@ -0,0 +1,62 @@
+# The Mission of the W3C
+
+The W3C leads the community in defining a World Wide Web that is inclusive and respectful of its users, 
```

Comment:
```suggestion
The W3C leads the community in defining technical standards and guidelines for a World Wide Web that is inclusive and respectful of its users, 
```
(this would give a clearer sense of what W3C is leading the community to do)

Reply:
rephrased in #111, please check there.